src/main/java/uk/gov/hmcts/darts/audio/mapper/GetTransformedMediaDetailsMapper.java

```diff
@@ -0,0 +1,67 @@
+package uk.gov.hmcts.darts.audio.mapper;
+
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.darts.audio.model.SearchTransformedMediaResponse;
+import uk.gov.hmcts.darts.audio.model.SearchTransformedMediaResponseCase;
+import uk.gov.hmcts.darts.audio.model.SearchTransformedMediaResponseCourthouse;
+import uk.gov.hmcts.darts.audio.model.SearchTransformedMediaResponseHearing;
+import uk.gov.hmcts.darts.audio.model.SearchTransformedMediaResponseMediaRequest;
+import uk.gov.hmcts.darts.common.entity.TransformedMediaEntity;
+
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class GetTransformedMediaDetailsMapper {
+    public List<SearchTransformedMediaResponse> mapSearchResults(List<TransformedMediaEntity> entityList) {
+        List<SearchTransformedMediaResponse> mappedDetails = new ArrayList<>();
+        for (TransformedMediaEntity entity : entityList) {
+            SearchTransformedMediaResponse transformedMediaDetails = new SearchTransformedMediaResponse();
```

Comment:
not a big deal, but i prefer to make another method that takes in the thing to be transformed (TransformedMediaEntity) and responds with the mapped response (SearchTransformedMediaResponse).
This helps to keep the complexity of the method down, instead of having a large for-loop